src/ClientOperations/AbstractSubscriptionOperation.php

```diff
@@ -91,7 +91,11 @@ public function __construct(
 
     protected function enqueueSend(TcpPackage $package): void
     {
-        ($this->getConnection)()->enqueueSend($package);
+        $connection = ($this->getConnection)();
+
+        if (null !== $connection) {
```

Comment:
This usually happens when you want to send something over wire (f.e. ack a message from subscription), but the connection was already closed. There is a very small time frame, where this race condition can occur. This solves the problem.

Reply:
Then I think we should add a comment about this into the code.

Reply:
go ahead, merge and add whatever comment you feel valuable

Reply:
okay will do back in my branch